x-pack/plugins/security_solution/public/common/components/multiselect_filter/index.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import { noop } from 'lodash';
+import type { EuiSelectableProps, FilterChecked } from '@elastic/eui';
+import { EuiPopover, EuiFilterButton, EuiSelectable } from '@elastic/eui';
+import { useBoolState } from '../../hooks/use_bool_state';
+
+export interface MultiselectFilterProps<T = unknown> {
```

Comment:
This component is a copy of [detection engine MultiselectFilte](https://github.com/elastic/kibana/blob/6d84c39f946b4e571dd6da01c22ff31d42e8fe8c/x-pack/plugins/security_solution/public/detection_engine/rule_monitoring/components/basic/filters/multiselect_filter/index.tsx#L6-L7).

The new version uses `EuiSelectable` instead of the deprecated `EuiFilterSelectItem`. I also had to export the filter without the `EuiFilterGroup` wrapper.
I duplicated the component because with `EuiSelectable`, we need to provide the `width` of the popover where previously it was dynamically calculated. 

That means we need two changes to migrate the existing components to use the new implementation:

1. Wrap it with `EuiFilterGroup`
```
<EuiFilterGroup>
  <MultiselectFilter ...
</EuiFilterGroup>
```

2. Provide the desired `width`
```
    <MultiselectFilter<RiskSeverity>
      ...
      width={150}
    />
```

Reply:
@machadoum Thanks for the clarification, can you please add the description to the code as well. Is there any ticket for tracking this?

Reply:
I liked the ticket to the PR, But the description is only an image. 😅

Reply:
Created a ticket for migration: https://github.com/elastic/kibana/issues/194301